ssl/t1_lib.c

```diff
@@ -1318,7 +1318,7 @@ TICKET_RETURN tls_decrypt_ticket(SSL *s, const unsigned char *etick,
         /* Some additional consistency checks */
         if (p != sdec + slen || sess->session_id_length != 0) {
```

Comment:
It's not like the pointer itself has changed just because what it pointed at has been deallocated. This should be fine.